File: Carthage/Checkouts/SwiftTheme/OCDemo/ChangeThemeCell.m
//
//  ChangeThemeCell.m
//  SwiftTheme
//
//  Created by Gesen on 16/5/27.
//  Copyright © 2016年 Gesen. All rights reserved.
//

#import "ChangeThemeCell.h"
#import "MyThemes.h"
#import "OCDemo-Swift.h"

@implementation ChangeThemeCell

- (void)awakeFromNib {
    [super awakeFromNib];
    
    [self.changeTheme theme_setTitleColor:[ThemeColorPicker pickerWithColors:@[@"#FFF", @"#000", @"#FFF", @"#000"]] forState:UIControlStateNormal];
    [self.changeTheme theme_setTitleColor:[ThemeColorPicker pickerWithColors:@[@"#555", @"#AAA", @"#555", @"#AAA"]] forState:UIControlStateHighlighted];
    self.changeTheme.theme_backgroundColor = [ThemeColorPicker pickerWithColors:@[@"#EB4F38", @"#F4C600", @"#56ABE4", @"#ECF0F1"]];
    self.changeTheme.layer.cornerRadius = self.changeTheme.bounds.size.width / 2;
}

- (IBAction)tapChange:(UIButton *)sender {
    [MyThemes switchToNext];
}

@end

File: Carthage/Checkouts/SwiftTheme/Demo/ChangeThemeCell.swift
//
//  ChangeThemeCell.swift
//  Demo
//
//  Created by Gesen on 16/3/2.
//  Copyright © 2016年 Gesen. All rights reserved.
//

import UIKit

class ChangeThemeCell: BaseCell {
    
    @IBOutlet weak var changeTheme: UIButton!

    override func awakeFromNib() {
        super.awakeFromNib()
        
        changeTheme.theme_setTitleColor(["#FFF", "#000", "#FFF", "#000"], forState: .normal)
        changeTheme.theme_setTitleColor(["#555", "#AAA", "#555", "#AAA"], forState: .highlighted)
        changeTheme.theme_backgroundColor = ["#EB4F38", "#F4C600", "#56ABE4", "#ECF0F1"]
        changeTheme.layer.cornerRadius = 60
    }
    
    @IBAction func tapChange(_ sender: UIButton) {
        MyThemes.switchToNext()
    }
    
    

}
